src/components/elements/CommonTruncatedList.tsx

```diff
@@ -12,11 +12,21 @@ const CommonTruncatedList: React.FC<Props> = ({ items }) => {
   const rest = items.slice(1);
 
   return (
-    <Box aria-label={items.join(', ')}>
+    <Box
+      display='flex'
+      flexDirection='row'
+      alignItems='center'
+      gap={0.5}
+      aria-label={items.join(', ')}
+    >
```

Comment:
Added center (vertical) alignment on the Box in order to avoid applying bottom margin directly to the Chip (removed below). We had added that to make this look nice in tables, but I wanted to remove it since it makes the non-table use case look worse.